content/automate/ManageIQ/System/Policy.class/transformationplanrequest_approved.yaml

```diff
@@ -0,0 +1,12 @@
+---
+object_type: instance
+version: 1.0
+object:
+  attributes:
+    display_name: 
+    name: TransformationPlanRequest_approved
```

Comment:
we should use the full name (`ServiceTemplateTransformationPlanRequest_approved` and the file name) everywhere under `/System`